envoy.yaml

```diff
@@ -0,0 +1,67 @@
+package:
+  name: envoy
+  version: 1.24.0 # TODO when upgrading to 1.25.0 remove all patches
+  epoch: 0
+  description: Cloud-native high-performance edge/middle/service proxy
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - git
+      - bazel-5
+      - openjdk-11
+      - bash
+      - libtool
+      - cmake
+      - samurai
+      - python3-dev
+      - clang-15
+      - libstdc++
+      - llvm15
+      - llvm15-dev
+      - llvm-lld
+      - coreutils
+      - patch
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/envoyproxy/envoy
+      branch: v${{package.version}}
+      destination: envoy
+
+  - runs: |
+      export JAVA_HOME=/usr/lib/jvm/openjdk
+      mkdir -p /builder/home/.cache/bazel/_bazel_root
+
+      cd envoy
+
+      # these patches are only required for 1.24.0, they are already merged upstream but 1.25.0 is not yet available.
```

Comment:
This is a shame but AFAICT necessary until 1.25.0 is available.  FWIW fetching [envoy](https://github.com/envoyproxy/envoy) `main` branch works with the same config but with ALL patches removed.  So this is untidy for now but we can't depend on a main upstream branch and forking for now doesn't seem sensible either.  So using the patch approach.

Reply:
It makes sense, even if I don't like it.